app/Models/Feed.php

```diff
@@ -355,7 +355,7 @@ public function loadEntries($feed) {
 				$this->id(),
 				$item->get_id(false, false),
 				$title === null ? '' : $title,
-				$author === null ? '' : html_only_entity_decode(strip_tags($author->name)),
+				$author === null ? '' : html_only_entity_decode(strip_tags($author->name === null ? $author->email : $author->name)),
```

Comment:
I would suggest `== ''` instead of `=== null` to catch other nullish values such as empty strings

Reply:
Good point, I pushed-force with a fix for that